de.metas.shipper.gateway.spi/src/main/java/de/metas/shipper/gateway/spi/model/DeliveryPosition.java

```diff
@@ -41,7 +41,8 @@
 	String content;
 	PackageDimensions packageDimensions;
 
-	/** optional, can be {@code null} */
+	/* optional, can be {@code null} */
+	@Nullable
```

Comment:
since u added `@Nullable` u can drop out the comment which says exactly the same thing.